android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -416,6 +470,9 @@ public void onHostDestroy() {
             localVideoTrack.release();
             localVideoTrack = null;
         }
+        if (android.os.Build.VERSION.SDK_INT >= 29) {
```

Comment:
Newline before the `if` statement:

```suggestion

        if (android.os.Build.VERSION.SDK_INT >= 29) {
```